tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "moduleResolution": "node",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
-    "target": "es2020",
+    "target": "ES2022",
```

Comment:
Are we pretty confident that everyone who visits Dockstore will have a browser that supports `ES2022`?
Related: https://compat-table.github.io/compat-table/es2016plus/
(https://github.com/tc39/proposal-regexp-match-indices is an issue for some browsers?)
(I copied this question from the PR that upgraded to Angular 14).

Reply:
Fair point, not sure, rolled back to es2020.

Reply:
Not sure if I'm reading the chart right, but it looks like ES2021 is as compatible as ES2020. 
Don't know if its worth it

Reply:
> Not sure if I'm reading the chart right, but it looks like ES2021 is as compatible as ES2020. Don't know if its worth it

Agreed, it looks the same, but I think I'll keep it to be safe.